lib/src/command/run.dart

```diff
@@ -86,4 +91,98 @@ class RunCommand extends PubCommand {
     });
     await flushThenExit(exitCode);
   }
+
+  /// Implement a v2 mode for use in `dartdev run`.
+  ///
+  /// Usage: `dartdev run [package[:command]]`
+  ///
+  /// If `package` is not given, defaults to current root package.
+  /// If `command` is not given, defaults to name of `package`.
+  /// If neither `package` or `command` is given and `command` with name of
+  /// the current package doesn't exist we fallback to `'main'`.
+  ///
+  /// Runs `bin/<command>.dart` from package `<package>`, with snapshot stored
+  /// in `.dart_tool/pub/bin/<package>/<command>.dart.snapshot.dart2`.
```

Comment:
Is this comment incomplete, or does this also want to run as a snapshot for the local package or a path dependency?